src/content/en/2022/markup.md

```diff
@@ -1,22 +1,564 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Markup
-#TODO - Review and update chapter description
-description: Markup chapter of the 2022 Web Almanac covering the use of elements, attributes, document trends, as well as adoption rates of new standards.
-authors: [AlexLakatos, ibnesayeed, daKmoR]
-reviewers: [j9t, bkardell]
+description: Markup chapter of the 2022 Web Almanac covering document data (doctypes, compression, languages, HTML coformance, document size), the use of HTML elements and attributes, data attributes and social media.
+authors: [j9t]
+reviewers: [tunetheweb, bkardell]
 analysts: [rviscomi]
-editors: []
+editors: [tunetheweb]
 translators: []
+j9t_bio: Jens Oliver Meiert is a web developer and author (<a hreflang="en" href="https://leanpub.com/css-optimization-basics"><cite>CSS Optimization Basics</cite></a>, <a hreflang="en" href="https://leanpub.com/web-development-glossary"><cite>The Web Development Glossary</cite></a>), who works as an engineering manager at <a hreflang="en" href="https://www.jimdo.com/">Jimdo</a>. He's an expert on web development specializing in HTML and CSS optimization. Jens contributes to technical standards and regularly writes about his work and research, particularly on his website, <a hreflang="en" href="https://meiert.com/en/">meiert.com</a>.
 results: https://docs.google.com/spreadsheets/d/1grkd2_1xSV3jvNK6ucRQ0OL1HmGTsScHuwA8GZuRLHU/
-featured_quote: TODO
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
+featured_quote: Without HTML there are no web pages, no web sites, no web apps. You can say that without HTML, there's no Web. That makes HTML one of the most important web standards, if not the most important web standard.
+featured_stat_1: 2.7%
+featured_stat_label_1: Documents using no doctype.
+featured_stat_2: 30 KB
+featured_stat_label_2: Median HTML document transfer size
+featured_stat_3: 29%
+featured_stat_label_3: Elements which are `div`s.
```

Comment:
@j9t I picked a quote and a few stats I thought might fit, but normally author's pick these so feel free to change as you see fit. [Here's what it looks like](https://20220923t160735-dot-webalmanac.uk.r.appspot.com/en/2022/?feat=markup#featured-chapter) with these:

<img width="621" alt="image" src="https://user-images.githubusercontent.com/10931297/191994830-f564e19e-afae-4dbd-9745-783968c345c3.png">

Reply:
Good idea [concentrating on the positive is one of our principles!](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#principles)

```suggestion
featured_quote: Without HTML there are no web pages, no web sites, no web apps. You can say that without HTML, there's no Web. That makes HTML one of the most important web standards, if not the most important web standard.
featured_stat_1: 90%
featured_stat_label_1: Documents using the HTML5 doctype.
featured_stat_2: 30 KB
featured_stat_label_2: Median HTML document transfer size
featured_stat_3: 29%
featured_stat_label_3: Elements which are `div`s.
```